apps/minaintyg/src/store/api.ts

```diff
@@ -12,7 +13,10 @@ export const api = createApi({
       query: () => 'user',
       providesTags: ['User'],
     }),
+    getCertificate: builder.query<{ content: CertificateListItem[] }, void>({
+      query: () => ({ url: 'certificates', method: 'POST' }),
```

Comment:
This endpoint is currently defined as /api/certificate in our backend. What do you think, should we go with certificate or certificates? :)

Reply:
"certificate" works fine 👍  I'll update the branch